### PR TITLE
README: clean up

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,12 +34,24 @@ curl -fsSL https://pkg.gpud.dev/install.sh | sh
...
+### Run GPUd with Lepton Platform
 
-For linux, run the following command to start the service (self-hosted option):
+Optionally you may register your machine with Lepton AI Platform -- the managed option brings several benefits:
+
+- Automated GPU health check and repair.
+- Centralized GPU metrics and logs.
+- Real-time GPU failure detection and alerting.
+
+> Please ensure that your machine has a public IP address and that the GPUd port (default 15132) is reachable.
```
Comment (xiang90) [inline README.md:45]: Remove this. this is not required anymore, right?

File: README.md [hunk 2]
```diff
@@ -34,12 +34,24 @@ curl -fsSL https://pkg.gpud.dev/install.sh | sh
 
 Note that the install script doesn't support other architectures (arm64) and OSes (macos), yet.
 
-### Run locally (self-hosted option)
+### Run GPUd with Lepton Platform
 
-For linux, run the following command to start the service (self-hosted option):
+Optionally you may register your machine with Lepton AI Platform -- the managed option brings several benefits:
```
Comment (xiang90) [inline README.md:39]: remove line39-line43 to make the readme short and clean.

File: README.md [hunk 3]
```diff
@@ -54,10 +66,25 @@ To check the logs of the running gpud:
 sudo gpud logs
 ```
 
-To access the local web UI, open https://localhost:15132 in your browser, as below:
+You can also start with the self-hosted option and later switch to the managed option:
```
Comment (xiang90) [inline README.md:69]: move this to the next section.

File: README.md [hunk 4]
```diff
@@ -54,10 +66,25 @@ To check the logs of the running gpud:
...
+# start without token
+sudo gpud up
+
+# when the token is ready, run the following command
+sudo gpud login --token <LEPTON_AI_WORKSPACE:TOKEM>
+```
+
+### Run GPUd standalone
+
+For linux, run the following command to start the service (self-hosted option):
```
Comment (xiang90) [inline README.md:81]: remove self-hosted mentioning.

File: README.md (hunk 3 above)
Comment (gyuho) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (gyuho) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (gyuho) [reply README.md]: done

**Comment by xiang90** [comment]
LGTM

**Review by xiang90 — APPROVED** [review]

**Pull request merged**